src/index.html

```diff
@@ -1,16 +1,16 @@
-<!DOCTYPE html>
-<html>
-	<head>
-		<meta charset="UTF-8">
-
-		<title>Solitaire</title>
-	</head>
-
-	<body>
-		<div class="app-container js-app-container">
-			<canvas class="app-canvas js-app-canvas">
-				Your browser does not support canvas elements.
-			</canvas>
-		</div>
-	</body>
+<!doctype html>
```

Comment:
Although the spec says it is case-insensitive[0], I am used to seeing `DOCTYPE` and that seems to be the preferred case[0].
I would prefer to keep `DOCTYPE`.
Why was this changed to lowercase?

[0] https://stackoverflow.com/questions/7020961/uppercase-or-lowercase-doctype